client/state/login/selectors.js

```diff
@@ -88,7 +82,7 @@ export const isTwoFactorEnabled = ( state ) => {
  * @returns {boolean}         Whether a request to authenticate 2FA is being made.
  */
 export const isRequestingTwoFactorAuth = ( state ) => {
-	return get( state, 'login.isRequestingTwoFactorAuth', false );
+	return state.login.isRequestingTwoFactorAuth ?? false;
```

Comment:
Here the state is always a valid boolean and doesn't need a fallback.

Reply:
Done.